### PR TITLE
add authorized directive

### PR DESCRIPTION
**Pull request by arinconstrio**

### CODE REVIEW

File: sample/src/main/scala/com/stratio/spray/oauth2/sample/ApiActor.scala [hunk 1]
```diff
@@ -20,25 +20,29 @@ import com.stratio.spray.oauth2.client.{OauthClient, OauthClientHelper}
...
+  import OauthClientHelper._
+
+  val myRoutes = {
+    {
       path("p1") {
-        authorize(hasRole(Seq("*"), user)) {
-          get {
-            complete(s"inn at  p1 ")
-          }
+        //        authorize(hasRole(Seq("*"), user)) {
```
Comment (josegom) [inline sample/src/main/scala/com/stratio/spray/oauth2/sample/ApiActor.scala]: why dont delete this line?

**Comment by anistal** [comment]
+1

**Comment by sgomezg** [comment]
+1

**Comment by sgomezg** [comment]
:shipit:

**Comment by stratiocommit** [comment]
Comment accepted. Will treat it ASAP

**Comment by stratiocommit** [comment]
Squashing and checks in progress.

**Comment by mpenate** [comment]
:shipit:

**Comment by stratiocommit** [comment]
Comment accepted. Will treat it ASAP

**Comment by stratiocommit** [comment]
Squashing and checks in progress.

**Pull request merged**